cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -200,22 +203,49 @@ public Long trigger(SlangTestCase testCase, CompilationArtifact compilationArtif
             return executionId;
         }
 
-        if(StringUtils.isNotBlank(errorMessageFlowExecution)){
+        if (StringUtils.isNotBlank(errorMessageFlowExecution)){
             // unexpected exception occurred during flow execution
             message = "Error occurred while running test: " + testCaseName + " - " + testCase.getDescription() + "\n\t" + errorMessageFlowExecution;
             log.info(message);
             throw new RuntimeException(message);
         }
 
+        String executionResult = executionReturnValues.getResult();
         if (result != null && !result.equals(executionResult)){
             message = TEST_CASE_FAILED + testCaseName + " - " + testCase.getDescription() + "\n\tExpected result: " + result + "\n\tActual result: " + executionResult;
             log.error(message);
             throw new RuntimeException(message);
         }
+
+        Map<String, Serializable> executionOutputs = executionReturnValues.getOutputs();
```

Comment:
Do you mean about the arrival of the Score & Slang events?
If so, I hope I fixed it by setting in the listener to flowFinished in case of proper termination, only according to the Slang: EVENT_EXECUTION_FINISHED event.
In case of error, I address also the score error events...